tensorboard/webapp/metrics/store/metrics_selectors_test.ts

```diff
@@ -610,6 +610,18 @@ describe('metrics selectors', () => {
       expect(selectors.getMetricsXAxisType(state)).toBe(XAxisType.WALL_TIME);
     });
 
+    it('returns hideEmptyCards when getMetricsHideEmptyCards is called', () => {
+      selectors.getMetricsHideEmptyCards.release();
+      const state = appStateFromMetricsState(
+        buildMetricsState({
+          settings: buildMetricsSettingsState({
+            hideEmptyCards: false,
+          }),
+        })
+      );
+      expect(selectors.getMetricsHideEmptyCards(state)).toBe(false);
```

Comment:
Although I recognize that the other boolean settings tests are written this way I have to say this doesn't give me a lot of confidence that the selector is actually returning hideEmptyCards value. Can you add a second case where you set hideEmptyCards: true and verify the selector returns true?